### PR TITLE
Add news page

### PR DESCRIPTION
**Pull request by leexgh**
Fix: https://github.com/genome-nexus/genome-nexus/issues/568
image

### CODE REVIEW

File: src/page/News.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import remarkGfm from 'remark-gfm';
+import * as React from 'react';
+import './News.scss';
+import $ from 'jquery';
+
+@observer
+class News extends React.Component<{}> {
+
+    readonly source = remoteData<string>(async () => {
+        return await $.get("https://raw.githubusercontent.com/leexgh/genome-nexus-frontend/news-page/src/doc/News.md");
```
Comment (leexgh) [inline src/page/News.tsx:15]: Will change to `https://raw.githubusercontent.com/genome-nexus/genome-nexus-frontend/master/src/doc/News.md` after review

File: src/page/News.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import remarkGfm from 'remark-gfm';
+import * as React from 'react';
+import './News.scss';
+import $ from 'jquery';
+
+@observer
+class News extends React.Component<{}> {
+    readonly source = remoteData<string>(async () => {
+        return await $.get(
+            'https://raw.githubusercontent.com/leexgh/genome-nexus-frontend/news-page/src/doc/News.md'
```
Comment (leexgh) [inline src/page/News.tsx:15]: Will change to `https://raw.githubusercontent.com/genome-nexus/genome-nexus-frontend/master/src/doc/News.md` after review

File: src/doc/News.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
+## Dec 16, 2021
+
+-   **API Update**: New search endpoint to search Genome Nexus database.
+    -   For example, search [BRAF p.V600E](https://www.genomenexus.org/search?keyword=BRAF%20p.V600E)
+
+## Nov 04, 2021
+
+-   **Website New Feature**: Add UniProt topology track to variant page. \* [Example: 7:g.55249071C>T](https://www.genomenexus.org/variant/7:g.55249071C%3ET)
```
Comment (inodb) [inline src/doc/News.md:8]: Maybe just remove `\*` here and in the other places? It looks a bit like a footnote indicator
```suggestion
-   **Website New Feature**: Add UniProt topology track to variant page. [Example: 7:g.55249071C>T](https://www.genomenexus.org/variant/7:g.55249071C%3ET)
```

**Review by inodb — APPROVED** [review]
LGTM!

File: src/doc/News.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    -   [Example: 7:g.55249071C>T](https://www.genomenexus.org/annotation/7:g.55249071C%3ET?fields=annotation_summary)
+
+## Mar 03, 2019
+
+-   **Command Line Interface Update**: Add OncoKB annotation to output annotation MAF file.
+
+## Feb 04, 2020
+
+-   **API Update**: Add [OncoKB](https://www.oncokb.org/) to annotation endpoints.
+    -   Please get your license fron [OncoKB](https://www.oncokb.org/) first and paste your `token` to API.
```
Comment (onursumer) [inline src/doc/News.md:88]:
```suggestion
    -   Please get your license from [OncoKB](https://www.oncokb.org/) first and paste your `token` to API.
```

File: src/doc/News.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+    <img src="https://user-images.githubusercontent.com/16869603/148015718-c0cfe2bb-dc6a-4a7d-b4b6-71a65e3e635a.png" />
+
+## Sep 18, 2019
+
+-   **Website New Feature**: Add variant page. \* [Example: 7:g.55249071C>T](https://www.genomenexus.org/variant/7:g.55249071C%3ET)
+    <img src="https://user-images.githubusercontent.com/16869603/148015822-887a52f4-bfa3-4ae9-b00d-7e27aff90bd3.png" />
+
+## Aug 15, 2019
+
+-   **API Update**: Add an API end point for MSK Insight (SIGNAL) mutations.
```
Comment (onursumer) [inline src/doc/News.md:117]:
```suggestion
-   **API Update**: Add an API endpoint for MSK Insight (SIGNAL) mutations.
```

**Review by onursumer — APPROVED** [review]
A couple of typos. Looks good otherwise!

File: src/doc/News.md (hunk 3 above)
Comment (leexgh) [reply src/doc/News.md]: @inodb This is added by prettier, so I add `md` file to `.prettierignore`

**Pull request merged**